tests/src/Kernel/Paragraphs/ContentRowInpageNavigationTest.php

```diff
@@ -0,0 +1,197 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_bootstrap_theme\Kernel\Paragraphs;
+
+use Drupal\paragraphs\Entity\Paragraph;
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * Tests the "Content row" paragraph, "Inpage navigation" variant.
+ */
+class ContentRowInpageNavigationTest extends ParagraphsTestBase {
+
+  /**
+   * Tests the rendering of the paragraph type.
+   */
+  public function testRendering(): void {
+    $paragraph_fact = [
+      0 => Paragraph::create([
+        'type' => 'oe_fact',
+        'field_oe_icon' => 'box-arrow-up',
+        'field_oe_title' => '1529 JIRA Ticket',
+        'field_oe_subtitle' => 'Jira Tickets',
+        'field_oe_plain_text_long' => 'Nunc condimentum sapien ut nibh finibus suscipit vitae at justo. Morbi quis odio faucibus, commodo tortor id, elementum libero.',
+      ]),
+      1 => Paragraph::create([
+        'type' => 'oe_fact',
+        'field_oe_icon' => 'box-arrow-up',
+        'field_oe_title' => '337 Features',
+        'field_oe_subtitle' => 'Feature tickets',
+        'field_oe_plain_text_long' => 'Turpis varius congue venenatis, erat dui feugiat felis.',
+      ]),
+      2 => Paragraph::create([
+        'type' => 'oe_fact',
+        'field_oe_icon' => 'box-arrow-up',
+        'field_oe_title' => '107 Tests',
+        'field_oe_subtitle' => 'Test tickets',
+        'field_oe_plain_text_long' => 'Cras vestibulum efficitur mi, quis porta tellus rutrum ut. Quisque at pulvinar sem.',
+      ]),
+    ];
+
+    $paragraph_accordion = [
+      0 => Paragraph::create([
```

Comment:
numerotation like this is redundant because the keys you add are the same as default from php 0, 1, 2 ..